File: 29a/Trainer.py
from Dipendenti import Dipendenti
class Trainer(Dipendenti):
    def __init__(self, cod, tipo, nome, costoOrario,oreSettimanali,specialita) -> None:
        super().__init__(cod, tipo, nome, costoOrario)
        self.oreSettimanali = oreSettimanali
        self.specialita     = specialita

    def __str__(self) -> str:
        return self.nome+"\t"+str(self.cod)+"\t"+self.tipo+"\t"+str(self.oreSettimanali)+"\t"+self.specialita+"\t"+"-"+"\t"+"-"+"\t"+"-\t"+str(self.costoOrario)

File: Sim05 -Cliente Dipendente Nutrizionista Trainer/src/Trainer.java
public class Trainer extends Dipendente {
    int ore_settimanali;
    String specialita;

    public Trainer(int codice, double costo_orario, String tipo, String nome_dipendente, int ore_settimanali,
            String specialita) {
        super(codice, costo_orario, tipo, nome_dipendente);
        this.ore_settimanali = ore_settimanali;
        this.specialita = specialita;
    }

    public String toString(){
        return nome_dipendente +" "+ codice+" " + tipo+" " + ore_settimanali+" " + specialita+ " - "+ " - "+" - "+ costo_orario; 
    }
}
